src/cpp/server/ServerDatabaseMigrationTests.cpp

```diff
@@ -48,6 +55,48 @@ std::map<DatasetVersion, std::map<DatasetType, std::string>> datasetPath {
             {Sqlite, "db/test/prairie-trillium-2022.02.2-485.sqlite.sql"},
             {Postgres, "db/test/prairie-trillium-2022.02.2-485.postgresql"}
         }
+    },
+    {
+        SpottedWakerobin, {
+            {Sqlite, "db/test/spotted-wakerobin-2022.07.2-576.sqlite.sql"},
+            {Postgres, "db/test/spotted-wakerobin-2022.07.2-576.postgresql"}
+        }
+    },
+    {
+        ElsbethGeranium, {
```

Comment:
This test also fails to build on open source.

Reply:
https://github.com/rstudio/rstudio/blob/bcf0247b47426e9ad750aa8982d9d36a482448e1/src/cpp/server/ServerDatabaseDataset.hpp#L7-L8

vs.

https://github.com/rstudio/rstudio-pro/blob/5e6bb779c4e52f7eac299d2d1d313fe1328c39d1/src/cpp/server/ServerDatabaseDataset.hpp#L7-L12

Reply:
Yeah that makes sense. I wasn't sure what the story is there. The pro repo has many more options then OS. Maybe we just stopped back-porting these changes?